faq.md

```diff
@@ -61,6 +62,22 @@ Request compressRequest(Request request) {
 Future<Response> postRequest(@Body() Map<String, String> data);
 ```
 
+## Runtime baseUrl change
+
+Sometimes you may need to change runtime base url, for example, you have server selector and you want to use different api routes like
```

Comment:
I would rephrase this line like this:

You may need to change the base URL of your network calls during runtime, for example, if you have to use different servers or routes dynamically in your app in case of a "regular" or a "paid" user.

Reply:
Thanks! Pushed your variant.